src/Components/Molecules/FMViewPicture/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { memo, useState } from "react";
+import { Box } from "@mui/material";
+
+import { IFMViewPicture } from "./FMViewPicture.interface";
+import { FAAvatar, FAButton } from "@Atoms";
+import { FMBackdropImage } from "@Molecules/FMBackdropImage";
```

Comment:
Kindly update the import order. 

```suggestion
import React, { memo, useState } from "react";
import Box from "@mui/material/Box";

import { FAAvatar, FAButton } from "@Atoms";
import { FMBackdropImage } from "@Molecules";

import { IFMViewPicture } from "./FMViewPicture.interface";
```